### ISSUE TITLE
Breadcrumbs: questionable use of hover

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#54747]
1. Hovering in the breadcrumbs tree usualy duplicates the name of field and adds in the brackets something like (property). I argue this is not useful and we shuold be consistent with the suggest widget and not show a hover here. People are already familiar with our suggest widget icons, and the hover brings no other useful value
2. Hovering over the the actual breadcrumbs I get the path hover (feels like duplication since we show the actual path) and over the names of classes duplicates the class name. I find this only useful when there is limited horizontal space.

**Comment by jrieken** [comment]
Yeah, that's a leftover from the shrink experiment... I'll remove for now
